test.sh

```diff
@@ -124,8 +124,8 @@ case $ENVIRONMENT_INDEX in
         rc=$?
         ;;
     4)
-        # go1.11
-        run_test_docker Dockerfile_build_ubuntu64_go1.11 test_64bit --test --junit-report
+        # go1.12
+        run_test_docker Dockerfile_build_ubuntu64_go1.12 test_64bit --test --junit-report
```

Comment:
I'm not sure if this is needed now, or if we should be using 1.13beta for this. The previous stages should be testing with 1.12 already so this seems like duplicate work.

Reply:
This line should be updated to 1.13beta along with the [release scripts](https://github.com/influxdata/influxdb/blob/1.8/releng/_go_versions.sh#L5). Go 1.13beta won't be out until August 1st though.

Reply:
1.13beta1 has already been released: https://groups.google.com/forum/?pli=1#!topic/golang-dev/Qn7Gc9-y-nw

1.13beta images appear to be available on Docker hub.